di_website/templates/publications/publication_page.html

```diff
@@ -74,4 +74,11 @@ <h2 class="type-m type-s--trailer">{{ page.download_report_title }}</h2>
         {% endif %}
     </div>
 </div>
+
+{% if page.publication_related_links|length > 0 %}
```

Comment:
I wonder if this had the same requirement as the blogs, events & news ... if so, then it'd need more than this to load at least 3.

Reply:
Some pages show related links. Some don't. So I assumed it was an optional thing.